asllib/instrumentation.ml

```diff
@@ -79,16 +87,15 @@ module Rule = struct
       ESlice;
       ECall;
       ERecord;
-      GetRecordField;
-      GetBitFields;
-      GetBitField;
+      EGetBitField;
+      EGetBitFields;
```

Comment:
I think that you should also add the instrumentation rules that you've added to this list:
```suggestion
      EGetBitFields;
      EUnknown;
      EPattern;
      EGetArray;
      ECondSimple;
      EUndefIdent;
```